david.css

```diff
@@ -1,64 +1,386 @@
+/* Reset og border-box á allt */
+* {
+	margin: 0;
+	padding: 0;
+}
+
+html {
+	box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+	box-sizing: inherit;
+}
 
+/* 						font						 */
 
+/* logo font er Lora, 700 */
+@font-face {
+	font-family: "Lora";
+	font-style: normal;
+	font-display: swap;
+	src: url(./fonts/Lora/Lora-VariableFont_wght.ttf);
+}
 
+/* restin er Lato 400 */
+@font-face {
+	font-family: "Lato";
+	src: url(./fonts/Lato/Lato-Regular.ttf);
+}
+
+
+/* 						settings 						*/
 :root {
+	/* sizes */
 	--offset: 15px;
-	--gutter: 30px;
+	--border_radius: 5px;
+	--border_size: 1px;
+	--gap_size: 30px;
+	--font_size: 16px;
+	--spacing: 1rem;
+	--logo_gap: 10px;
 	--columns: 12;
-    --grid-color: hsla(125, 67%, 51%, 0.25);
-    --grid-repeating-width: calc(100% / var(--columns));
-    --grid-column-width: calc((100% / var(--columns)) - var(--gutter));
-    --grid-background-width: calc(100% + var(--gutter));
-    --grid-background-columns: repeating-linear-gradient(
-      to right,
-      var(--grid-color),
-      var(--grid-color) var(--grid-column-width),
-      transparent var(--grid-column-width),
-      transparent var(--grid-repeating-width)
-    );
+	--site_max_width: 1440px;
+	--grid-column-width: calc((100% / var(--columns)) - var(--gap_size));
+
+	/* colors */
+	--latte: #e1cec2;
+	--dark-brown: #3c2a1e;
+	--brown: #6f4e37;
+	--white: #ffffff;
+	--light: #FEFEFE;
+	--top-grey: #ecebec;
+	--top-grey2: #f2f1f3;
+
+	/* fonts */
+	--font-base-size: 1rem;
+	--font-family-base: 'Lato', sans-serif;
+	--font-family-logo: 'Lora', serif;
 }
 
-html {
-    position: relative;
-}
-  
-html::before {
-    content: '';
-  
-    position: absolute;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    left: 0;
+
+
+
+/* 						global						 */
+
+/** Hjálpar class sem felur sjónrænt en skjálesarar lesa **/
+.sr-only:not(:focus):not(:active) {
+	clip: rect(0 0 0 0);
+	clip-path: inset(50%);
+	height: var(--border_size);
+	overflow: hidden;
+	position: absolute;
+	white-space: nowrap;
+	width: var(--border_size);
+}
+
+html,
+input,
+select,
+button {
+	font-family: var(--font-family-base);
+}
+
+li {
+	font-size: var(--font_size);
+	list-style: none;
+}
+
+
+
+
+
+img {
+	width: 100%;
+}
+
+
+
+
+
+/* 				grid containter 				*/
+
+.top2bottom {
+	display: grid;
+	grid-template-areas:
+		"top"
+		"middle"
+		"bottom";
+	grid-template-columns: 1fr;
+	grid-template-rows: auto auto minmax(0, 270px);
+	align-items: center;
+	padding: 0 0;
+	justify-items: center;
+	align-items: flex-end;
+	justify-content: center;
+}
+
+.grid_container {
+	opacity: 1;
```

Comment:
if it's in styles.css it doesn't need to be in david.css, would be great if you could clean it up